mytoyota/models/endpoints/trips.py

```diff
@@ -23,6 +38,30 @@ class _SummaryBaseModel(BaseModel):
         alias="fuelConsumption", default=None
     )  # Electric cars might not use fuel. Milliliters.
 
+    def __add__(self, other: _SummaryBaseModel):
```

Comment:
Could you please explain what exactly this is needed for.
Please also add a few unit tests to maintain the coverage. I think these could also contribute to the explanation.